src/toolkit/position.cc

```diff
@@ -25,18 +27,40 @@ double Position::longitude() const {
 }
 
 void Position::latitude(double lat) {
+  LatCheck(lat);
   latitude_ = SetPrecision(lat * CYCLUS_DECIMAL_SECOND_MULTIPLIER, 1);
 }
 
 void Position::longitude(double lon) {
+  LonCheck(lon);
   longitude_ = SetPrecision(lon * CYCLUS_DECIMAL_SECOND_MULTIPLIER, 1);
 }
 
 void Position::set_position(double lat, double lon) {
+  LatCheck(lat);
+  LonCheck(lon);
   latitude_ = SetPrecision(lat * CYCLUS_DECIMAL_SECOND_MULTIPLIER, 1);
   longitude_ = SetPrecision(lon * CYCLUS_DECIMAL_SECOND_MULTIPLIER, 1);
 }
 
+void Position::LatCheck(double lat) {
+  if (lat > 90 || lat < -90){
+    std::stringstream msg;
+    msg << "The provided latitude (" << lat
+        << ") is outside the acceptable range.";
```

Comment:
The error message should include what the acceptable range is, as well as what the value passed in was.

Reply:
I agree. Thank you for the comment. I'll fix this right away!

Reply:
The value passed is currently included in the msg.
Thus, I've added the acceptable range using interval notation.